tests/cypress/integration/LSP/F95833.spec.js

```diff
@@ -16,45 +16,53 @@
 //@ts-ignore
 /// <reference types="../../support/" />
 
-// Import selectors from Theia object
-import { Theia } from '../../../node_modules/@eclipse/che-che4z/tests/dist/selectors';
+import { Theia, VSCODE } from '@eclipse/che-che4z/tests/dist/selectors';
+
+const env = Cypress.env('ide');
+const IDE = env === 'theia' ? Theia : VSCODE;
 
 //F95833 - LSP for COBOL - support for CICS statements (basic intellisense)
 
 context('This is F95833 spec', () => {
   describe('TC312735 Check EXEC CICS is in Procedure Division', () => {
     it(['smoke', 'CI'], 'Checks that EXEC CICS could be run *only* under Procedure Division.', () => {
       cy.openFile('ADSORT.cbl').goToLine(59);
-      cy.getCurrentLine().should('not.have.class', Theia.editorError);
+      cy.getCurrentLine().should('not.have.class', IDE.editorError);
       cy.getCurrentLine().type('{selectall}{backspace}');
       cy.goToLine(35);
       cy.getCurrentLine().type('           EXEC CICS XCTL PROGRAM (XCTL1) END-EXEC.').wait(500);
       cy.goToLine(35);
-      cy.getCurrentLineErrors({ expectedLine: 35 })
-        .getHoverErrorMessage()
-        .contains('Missing token SQL at execSqlStatement');
+      cy.getCurrentLineErrors({ expectedLine: 35 });
+      if (IDE === Theia) {
+        cy.getCurrentLineErrors({ expectedLine: 35 }).getHoverErrorMessage();
+      } else {
+        cy.getCurrentLineErrors({ expectedLine: 35 }).getHoverErrorMessage('CICS');
```

Comment:
Wait, what? Hovering error messages doesn't work on Theia?

Reply:
No worries, it is because of VSC and Cypress. I needed to use the extended library to hover over variables (use the real events instead of simulated ones by default). And I need to add specifically the part of the string to do that...